Chap_Terms.tex

```diff
@@ -20,23 +20,30 @@ \chapter{PMIx Terms and Conventions}
 The following terminology is used throughout this document:
 
 \begin{itemize}
-\item \declareterm{session} refers to a pool of resources with a unique identifier (a.k.a., the \emph{session ID}) assigned by the \ac{WLM} that has been reserved for one or more users. Historically, \ac{HPC} sessions have consisted of a static allocation of resources - e.g., a block of nodes assigned to a user in response to a specific request and managed as a unified collection. However, this is changing in response to the growing use of dynamic programming models that require on-the-fly allocation and release of system resources. Accordingly, the term \emph{session} in this document refers to a potentially dynamic entity, perhaps comprised of resources accumulated as a result of multiple allocation requests that are managed as a single unit by the \ac{WLM}.
 
-\item \declareterm{job} refers to a set of one or more \emph{applications} executed as a single invocation by the user within a session with a unique identifier (a.k.a, the \emph{job ID}) assigned by the \ac{RM} or launcher. For example, the command line ``\textit{mpiexec -n 1 app1 : -n 2 app2}'' generates a single \ac{MPMD} job containing two applications. A user may execute multiple \emph{jobs} within a given session, either sequentially or in parallel.
+\item \declareterm{session} 
+refers to a set of resources assigned by the \ac{WLM} that has been 
+reserved for one or more users. 
+A session is identified by a \emph{session ID} that is  
+a unique within the scope of the governing \acp{WLM}.
+Historically, \ac{HPC} sessions have consisted of a static allocation of resources - i.e., a block of resources assigned to a user in response to a specific request and managed as a unified collection. However, this is changing in response to the growing use of dynamic programming models that require on-the-fly allocation and release of system resources. Accordingly, the term \emph{session} in this document refers to a potentially dynamic entity, perhaps comprised of resources accumulated as a result of multiple allocation requests that are managed as a single unit by the \ac{WLM}.
 
```

Comment:
Notes from WG:  Use of pool was confusing as it made it sound indefinite whether they were really assigned or just available to be assigned.  I think I understand that they are a pool in the sense that they are available to satisfy the needs of job creation, but overall it was deemed clearer to avoid the use of "pool".  

Would prefer to get rid of expressions marked by "-", but did not do that yet in this case.  

"One or more users"... just want to make sure that is really what we want?